src/__tests__/recall-by-number.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, test } from '@jest/globals'
+import { parsePresetVariableOption } from '../recall-by-number.js'
+import { CompanionOptionValues } from '@companion-module/base'
+import { PresetRecall } from '../camera/commands.js'
+import { prettyBytes } from '../visca/utils.js'
+import { MockContext } from './mock-context.js'
+
+describe('test recall preset values', () => {
+	test('Variable value contant used dec 37', async () => {
+		const context = new MockContext()
+		const input: CompanionOptionValues = { val: '37' }
```

Comment:
Maybe add a global

```ts
function optionsWithRecallPresetOption(text: string): CompanionOptionValues {
    return { val: text }
}
```

and use that, to repeat the type and `val` only in one spot rather than all of them .

Also, can you rename `input` to `options`, seeing as that's what it's called everywhere else in code?  `input` confused me for a second there.